plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -351,6 +351,9 @@ def _decide_drpms_to_download(self, metadata_files):
             _logger.debug('skipping DRPM sync')
             return set(), 0, 0
         presto_file_handle = metadata_files.get_metadata_file_handle(presto.METADATA_FILE_NAME)
+        if not presto_file_handle:
+            presto_file_handle = metadata_files.get_metadata_file_handle(
+                presto.ALTERNATE_METADATA_FILE_NAME)
```

Comment:
Should get_metadata_file_handle raise an Exception if it doesn't exist?

Also, a comment explaining this would be nice.